version/docs/version.asciidoc

```diff
@@ -1,4 +1,4 @@
-:stack-version: 8.17.1
+:stack-version: 8.17.2
```

Comment:
We should check if this even does anything, would be an automation PR we could just remove.